Tools/clinic/libclinic/_overlong_docstrings.py

```diff
@@ -0,0 +1,433 @@
+OVERLONG_SUMMARY = frozenset((
```

Comment:
Another possible approach is to add a decorator that will set the flood for the function. This will keep it closer to the function definition, but it will be harder to get a list of all the flagged functions.

I don't know which approach is better, I'm just showing an alternative possibility.